client/auth/index.js

```diff
@@ -10,8 +10,9 @@ import config from 'config';
 import controller from './controller';
 
 export default () => {
+	page( '/oauth-login', controller.oauthLogin );
```

Comment:
I mean, it's misleading that we declare a controller but the redirect is actually because something else and we don't use the controller at all